scripts/prepare.js

```diff
@@ -12,7 +12,7 @@ if (!fs.existsSync("analyzers")) {
 }
 
 downloadIfNeeded(
-  "https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/3.1.0.1376/sonarlint-language-server-3.1.0.1376.jar",
+  "https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/3.5.0.1548/sonarlint-language-server-3.5.0.1548.jar",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace double quotes by single quotes ([javascript:SingleQuote](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ASingleQuote))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=15&issues=AWNdSse_nxz_2987348L&open=AWNdSse_nxz_2987348L)

[ISSUE_KEY]: AWNdSse_nxz_2987348L